torchrl/objectives/ppo.py

```diff
@@ -87,6 +88,8 @@ class PPOLoss(LossModule):
             Can be one of "l1", "l2" or "smooth_l1". Defaults to ``"smooth_l1"``.
         normalize_advantage (bool, optional): if ``True``, the advantage will be normalized
             before being used. Defaults to ``False``.
+        normalize_advantage_exclude_dims (Sequence[int], optional): dimensions to exclude from the advantage
```

Comment:
```suggestion
        normalize_advantage_exclude_dims (Tuple[int], optional): dimensions to exclude from the advantage
```

Reply:
This seems like an indirect argument to me and it could be hard for users to understand.
If you're using PPO with this kwarg, you need to think this way:
1. I asked to normalize the advantage
2. My reward has one or more agent dims
3. I want to excldue these from the advantage normalization.
Another way to do this could be to have a kwarg
```
reward_agent_dim: int | Tuple[int] | None = None,
```
If this is passed, the `reward_agent_dim` will be kept during calls to `mean` / `std`. 
Per se, it changes nothing but it's a tiny bit more clear from the name and signature that this indicates a dim that is not a feature dim.

Reply:
See my discussion point on this being useful for more uscases. We can come up with a better name together